nui/app.js

```diff
@@ -0,0 +1,27 @@
+const wrapper = document.getElementById("root")
+
+const sendMessageToFrame = (iframe, message) => iframe.contentWindow.postMessage(message, "*")
+
+window.addEventListener("message", (evt) => {
+    const currentFrame = document.getElementById('bankingFrame')
```

Comment:
This is temporary until we build a completely new UI.
It allows us to keep both UI's in one resource without a complete refactor.
It's not ideal, but it works very well.